include/zephyr/bluetooth/gatt.h

```diff
@@ -172,7 +172,7 @@ struct bt_gatt_attr {
 	void *user_data;
 	/** Attribute handle */
 	uint16_t handle;
-	/** Attribute permissions */
+	/** Attribute permissions, always 0 */
```

Comment:
That is not true. The `attr->perm` is a bitfield of the `BT_GATT_PERM_` values.

Reply:
Apologies that was my mistake. I should have made the comment clearer.
"The permissions of a given attribute are defined by a higher layer specification, and
are not discoverable using the Attribute protocol." BT Spec 5.3 Vol 3 Part F 3.2.5 so it should always be 0 *if returned by bt_gatt_discover()*. This behavior was reported in confusion as to why a 0 value was always received for attribute permissions after calling bt_gatt_discover() so adding documentation in an effort to avoid future confusion for others that its the intended behavior.

Reply:
Indeed. We should document that the `attr->perm` is 0 when doing discovery (in fact I believe it would probably be 0 in many other cases as well, but a bit unsure...)